templates/iot/examples/iot-project-managed.yaml

```diff
@@ -1,12 +1,12 @@
 kind: IoTProject
 apiVersion: iot.enmasse.io/v1alpha1
 metadata:
-  name: iot
+  name: iot2
```

Comment:
Why change to iot2? Is it clashing with something?

Reply:
:+1: Shouldn't have been pushed. Will fix.